source/Generic/PlayState/Converters/IconPathConverter.cs

```diff
@@ -10,19 +10,22 @@
 
 namespace PlayState.Converters
 {
-    public class SuspendModeToStringConverter : MarkupExtension, IValueConverter
+    public class IconPathConverter : MarkupExtension, IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
         {
-            var source = (SuspendModes)value;
-            switch (source)
+            var fullFilePath = value != null && !((string)value).IsNullOrEmpty() ? API.Instance.Database.GetFullFilePath((string)value) : null;
```

Comment:
I think it should be checked if the image file exists in the obtained path

Reply:
I thought about that but I recall that you told me that images could also be URLS, that's why I didn't add it.

Reply:
It's possible to detect if it's an URL by checking if the string starts with `http`, then only check the files that are local files

Reply:
If you want to skip icons that are url then that's also fine for me